tests/helper_functions.py

```diff
@@ -26,30 +26,36 @@ def random_file(ext):
 
 
 def score_model_in_sagemaker_docker_container(
-        model_path, data, content_type, flavor=mlflow.pyfunc.FLAVOR_NAME):
+        model_path, data, content_type, flavor=mlflow.pyfunc.FLAVOR_NAME,
+        activity_polling_timeout_seconds=500):
     """
     :param model_path: Path to the model to be served.
     :param data: The data to send to the docker container for testing. This is either a
                  Pandas dataframe or string of the format specified by `content_type`.
     :param content_type: The type of the data to send to the docker container for testing. This is
                          one of `mlflow.pyfunc.scoring_server.CONTENT_TYPES`.
     :param flavor: Model flavor to be deployed.
+    :param activity_polling_timeout_seconds: The amount of time, in seconds, to wait before
+                                             declaring the scoring process to have failed.
     """
     env = dict(os.environ)
     env.update(LC_ALL="en_US.UTF-8", LANG="en_US.UTF-8")
     proc = _start_scoring_proc(
             cmd=['mlflow', 'sagemaker', 'run-local', '-m', model_path, '-p', "5000", "-f", flavor],
             env=env)
-    return _evaluate_scoring_proc(proc, 5000, data, content_type)
+    return _evaluate_scoring_proc(proc, 5000, data, content_type, activity_polling_timeout_seconds)
 
 
-def pyfunc_serve_and_score_model(model_path, data, content_type):
+def pyfunc_serve_and_score_model(
+        model_path, data, content_type, activity_polling_timeout_seconds=500):
```

Comment:
Good question! I bumped up the number because I was finding that the conda environment installation process for PyTorch takes a significant amount of time: it needs to install the `torchvision` libraries which depend on cudNN. The installation process usually did not finish within 250 seconds. However, 360 seconds should be just fine. I've updated the timeout to `360`. Let me know if this works for you!